cloud-compose/cloud-compose.yml.example

```diff
@@ -39,3 +39,41 @@ cluster:
         - ${CLUSTER_SUBNET_0}
         - ${CLUSTER_SUBNET_1}
         - ${CLUSTER_SUBNET_2}
+datadog:
+  name: ${CLUSTER_NAME}
+  notify: 
+    - '@slack-arc-platform-ops'
+    - '@opsgenie-platform-tools'
+  options:
+    notify_no_data: true
+    no_data_timeframe: 480
+  monitors:
+    - 
+      tag: high-cpu-load
+      message: 'The CPU load is too high on server {{host.name}}.'
+      name: 'High CPU Load'
+      query: 'avg(last_4h):avg:system.cpu.idle{clustername:%(name)s} by {host} < 25'
+    -
+      tag: low-${VOLUME}-disk-space
```

Comment:
can you add an example for the root and data disk space instead of making that configurable by volume? In most cases everyone will have the same root volume monitor and some people will delete the data volume monitor. Put the data volume one in, but comment it out so it won't run unless you enable it.